### ISSUE TITLE
SpaceBetweenParens raised unexpected error / undefined method `[]' for nil:NilClass'

### ISSUE THREAD

**Issue by s10wen** [brigade/scss-lint#739]
Wondering if anyone can point me in the right direction why this error would occur for the following code;

```
$default-font-family : "Helvetica Neue", Helvetica, Arial, "Hiragino Sans GB", "冬青黑体", STHeiti, "华文黑体", "Microsoft YaHei", "微软雅黑", sans-serif;
```

I'm guessing `scss-lint` may not be recognising the Chinese characters?

**Comment by sds** [comment]
Hey @s10wen,

I can't reproduce your issue. When I paste the above code and run it against the latest version of scss-lint (0.47.0 at time of writing), it correctly reports:

```
test.scss:1 [W] SpaceAfterVariableName: Variable names should be followed immediately by a colon
```

Are you running the latest version of SCSS-Lint?

**Comment by s10wen** [comment]
Thanks @sds turns out I wasn't on the latest release sorry! I saw: https://rubygems.org/gems/scss-lint/versions/0.38.0 and thought the latest release was `0.38.0` I've updated to `0.47.1` and it works now, thanks.
